dotcom-rendering/src/web/components/NewslettersPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { css, Global } from '@emotion/react';
+import { brandAlt, focusHalo, neutral } from '@guardian/source-foundations';
+import { StrictMode } from 'react';
+import type { NavType } from '../../model/extract-nav';
+import type { DCRNewslettersPageType } from '../../types/newslettersPage';
+import { NewslettersPageLayout } from '../layouts/NewslettersPageLayout';
+import { AlreadyVisited } from './AlreadyVisited.importable';
+import { FocusStyles } from './FocusStyles.importable';
+import { Island } from './Island';
+import { Metrics } from './Metrics.importable';
+import { SkipTo } from './SkipTo';
+
+type Props = {
+	newslettersPage: DCRNewslettersPageType;
+	NAV: NavType;
+};
+
+/**
+ * @description
+ * NewslettersPage is a high level wrapper for the newsletters page on Dotcom. Sets strict mode and some globals
+ *
+ * @param {Props} props
+ * @param {DCRNewslettersPageType} props.newslettersPage - The newsletters JSON data
+ * @param {NAVType} props.NAV - The nav JSON data
+ * */
+export const NewslettersPage = ({ newslettersPage, NAV }: Props) => {
+	return (
+		<StrictMode>
+			<Global
+				styles={css`
+					/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
+					/* The not(.src...) selector is to work with Source's FocusStyleManager. */
+					*:focus {
+						${focusHalo}
+					}
+					::selection {
+						background: ${brandAlt[400]};
+						color: ${neutral[7]};
+					}
+				`}
+			/>
+			<SkipTo id="maincontent" label="Skip to main content" />
+			<SkipTo id="navigation" label="Skip to navigation" />
+			<Island clientOnly={true} deferUntil="idle">
+				<AlreadyVisited />
+			</Island>
```

Comment:
is this required for the newsletters page?

Reply:
I'm not sure what `AlreadyVisited` is actually for - I left it in to keep the page format consistent, but if its only relevant to fronts and/or articles, we could remove it

Reply:
to do - ask data design (?) if they know what the 'gu.alreadyVisited' local storage value is for and if it shoudl be on this page  OR see if it's on the frontend version

Reply:
The equivalent frontend code is in [this script file](https://github.com/guardian/frontend/blob/cf9d238b45c8dde9d728666e73cbc49c31698173/static/src/javascripts/bootstraps/standard/main.js) - the comments say "This file is intended to be downloaded and run ASAP on all pages by all readers". It seems some of the commercial code reads the value.
Looks like it is required on non-article, non-front pages.